draft-ietf-core-senml.md

```diff
@@ -195,7 +195,7 @@ configured.
 Connecting sensors to the Internet is not new, and there have been
 many protocols designed to facilitate it. This specification defines a
 format and media types for carrying simple sensor information in a
-protocol such as HTTP {{?RFC7230}} or CoAP {{RFC7252}}.  This format
```

Comment:
Yep, this made me stop in my final review as well but it just was below threshold.  Thanks for fixing.

Reply:
No problem. Made Ben C stop too so certainly worth fixing.